vyper/context/types/abstract.py

```diff
@@ -16,16 +16,26 @@ def compare_type(self, other):
             pass
         return isinstance(other, type(self))
 
+    def __repr__(self):
+        value = super().__repr__()
+        if value.startswith("<"):
```

Comment:
Why does this mean it gives meaningful output?

Reply:
the opposite actually.  a normal repr is `<ClassName object at 0x...>`, whereas the desired ones will be e.g. "string".  If it opens with `<`, we know it's not implemented and so we use `_description`.

Reply:
Ah, could it be `f"<{self.__class__.__name__}"` instead, so we know that it's a normal Python repr?